Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -396,11 +397,38 @@ function Invoke-CodeGen {
     Invoke-Task -name $MyInvocation.MyCommand.Name -task {
         $tool = (Join-Path $toolsPath 'EdFi.Ods.CodeGen')
         $repositoryRoot = (Get-RepositoryRoot $implementationRepo).Replace($implementationRepo, '')
-
-        if ($IncludePlugins) {
+        $total=$ExtensionPaths.Split(" ").Length
+        $counter=0
+
+        if (-not [string]::IsNullOrWhiteSpace($ExtensionPaths)) {
+            $counter=1
+            Foreach ($extensionPath in $ExtensionPaths.Split(" ")) {
+
+                if (![System.IO.Directory]::Exists($extensionPath)) {
+                    throw "Unable to find extension Location project path at location $extensionPath ."
+                }
+
+                if($counter -eq $total){
+                    $extensionLocationPlugins += $extensionPath
+                }
+                else
+                {
+                    $extensionLocationPlugins += $extensionPath + ','
+                }
+                $counter++
+            }
+        }
+       
+        if ($IncludePlugins -and $counter -eq 0 ) {
             & $tool -r $repositoryRoot -e $Engine -IncludePlugins | Write-Host
         }
-        else {
+        elseif ($IncludePlugins -and $counter -gt 0 ) {
+            & $tool -r $repositoryRoot -e $Engine -IncludePlugins --ExtensionPaths $extensionLocationPlugins | Write-Host
+        }
+        elseif ($IncludePlugins -ne 'True' -and $counter -gt 0 ) {
```

Comment:
I think `$IncludePlugins -ne 'True'` works because `switch != string` will always be true. should we change these to `-not $IncludePlugins`?

```powershell
function Test-Switch([switch]$switch) { Write-Host ($switch -eq 'True') }

Test-Switch
# False

Test-Switch -switch
# False
```

Reply:
I fixed it . Thanks